portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/displaypagetemplate/DisplayPageTemplatesWithItemPreview.testcase

```diff
@@ -105,6 +105,65 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-131040. The category name should be shown when select a category in Content Display with Default Template."
+	@priority = "4"
+	test ViewCategoryNameInContentDisplayWithDefaultTemplate {
```

Comment:
Had better to rename to ContentDisplay#ViewSelectedCategoryInDisplayPageTemplate